docs/contributing/contributing-your-profile.md

```diff
@@ -0,0 +1,44 @@
+## A simple step by step guide with explanation for beginners to add their profile:
+
+- Fork it to your accont.
+- Now your repo name looks like this GITHUB_YOURUSERNAME/LinkFree.
+- Then in the top right corner you will get a Code option, click it. Then copy the SSH link.
+  [NOTE: If you haven't set your SSH key yet, Checkout this video.](https://www.youtube.com/watch?v=lV5mrUYsucU&t=4s)
+
+- Now turn on Git, choose your own folder/directory to proceed ahead. In my case i choosed Desktop.
+
+- Then type `git clone *The SSH link you copied*`, Press Enter.
+
+- Then get into the project folder through Git CLI. For that type `cd LinkFree` , press Enter.
+
+-Now you are into the LinkFree folder. Now type `git remote -v` ,press enter it will show your origin url.

```

Comment:
```suggestion
- Now you are into the LinkFree folder. Now type `git remote -v`, press enter it will show your origin url.
```
Fixed the line into proper format.